ext/pdo_odbc/odbc_driver.c

```diff
@@ -386,7 +410,9 @@ static int odbc_handle_get_attr(pdo_dbh_t *dbh, zend_long attr, zval *val)
 		case PDO_ODBC_ATTR_ASSUME_UTF8:
 			ZVAL_BOOL(val, H->assume_utf8 ? 1 : 0);
 			return 1;
-
+		case PDO_ATTR_AUTOCOMMIT:
+			ZVAL_LONG(val, dbh->auto_commit);
+			return 1;
```

Comment:
Shouldn't this be a bool?

Reply:
OCI returns bool, others return long. OCI has been decided to be unbundled, so I adapted it to specifications other than OCI.

But certainly...on the contrary, I may need to change the others to bool.

https://github.com/php/php-src/blob/40d1442cec40fbf96cb47ea1afbc1a715ee513a7/ext/pdo_mysql/mysql_driver.c#L534
https://github.com/php/php-src/blob/40d1442cec40fbf96cb47ea1afbc1a715ee513a7/ext/pdo_oci/oci_driver.c#L631
https://github.com/php/php-src/blob/40d1442cec40fbf96cb47ea1afbc1a715ee513a7/ext/pdo_firebird/firebird_driver.c#L998

Reply:
Yes, really...
There are other issues like this, so I'll try to fix them in another pull request.

For now, I'll fix this to bool.

Reply:
>For now, I'll fix this to bool.

done